control/server.py

```diff
@@ -58,6 +58,20 @@ def int_to_bitmask(n):
     """Converts an integer n to a bitmask string"""
     return f"0x{hex((1 << n) - 1)[2:].upper()}"
 
+def cpumask_set(args):
+    """Check if reactor cpu mask is set in command line args"""
+
+    # Check if "-m" or "--cpumask" is in the arguments
+    if "-m" in args or "--cpumask" in args:
```

Comment:
This is redundant

Reply:
OK. If we force the user not to use "" in the comf file then we can use "-m" in args. Still we don't need the "--cpumask" in args part. This is not a valid case, "--cpumask" must be followed by a "=".